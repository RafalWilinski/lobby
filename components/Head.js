import React from "react";
import Head from "next/head";

export default ({ children }) => (
  <div>
    <Head>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
      <meta charSet="utf-8" />
      <link
        rel="stylesheet"
        href="//cdnjs.cloudflare.com/ajax/libs/antd/2.9.3/antd.min.css"
      />
    </Head>
    {children}
  </div>
);
