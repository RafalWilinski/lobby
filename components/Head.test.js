import React from "react";
import renderer from "react-test-renderer";
import Head from "./Head";

describe("Head", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Head />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
