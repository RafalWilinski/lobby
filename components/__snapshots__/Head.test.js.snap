// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders correctly 1`] = `<div />`;
